tools/cockpit.spec

```diff
@@ -331,17 +331,22 @@ The Cockpit components for managing virtual machines.
 
 %files machines -f machines.list
 
-%package ovirt
+%package machines-ovirt
 Summary: Cockpit user interface for oVirt virtual machines
 Requires: %{name}-bridge >= %{required_base}
 Requires: %{name}-system >= %{required_base}
 Requires: libvirt
 Requires: libvirt-client
+# package of old name "cockpit-ovirt" was shipped on fedora only
+if 0%{?fedora} >= 25
+Provides: %{name}-machines-ovirt = %{version}-%{release}
```

Comment:
Providing your own name doesn't do anything. This should provide the old name so that dependencies to "cockpit-ovirt" don't break. However, I think we may not actually have any packages that depend on cockpit-ovirt - if not, then the Provides: isn't necessary at all, and the Obsoletes: is sufficient for upgrades. See https://fedoraproject.org/wiki/Upgrade_paths_%E2%80%94_renaming_or_splitting_packages

Reply:
Thanks for the link. Considering context, I can hardly imagine any non-experimental package depending on this one.